### PR TITLE
Solved problem 6.

### PR DESCRIPTION
**Pull request by paulaOliverosHollis**
I used a for loop to fing the sum of the squares of the first 100 natural numbers.

I used a second loop to add numbers from 1 to 100. Then I multiplied the total sum by itself.

Lastly, I substracted the sum of the squares from the square of the sum in order to get the difference.

### CODE REVIEW

File: ProjectEuler/Problems/Problems_001_To_025.cs [hunk 1]
```diff
@@ -141,11 +141,29 @@ private static bool IsDivisibleBy1Through20(int number)
         public static void Problem6()
         {
             Console.WriteLine("\n\n\n6. Find the difference between the sum of the squares of the first one hundred natural numbers and the square of the sum.");
+
+            long sumOfSquares = 0;
+
+            for(int i = 1; i <= 100; i++)
```
Comment (zhollis21) [inline ProjectEuler/Problems/Problems_001_To_025.cs]: I noticed you are doing 2 loops, but really you could do it twice as fast with just one loop.
`
for(int i = 1; i <= 100; i++)
{
     sumOfSquares += i * i;
     sumOfFirstHundredNums == i;
}

`

File: ProjectEuler/Problems/Problems_001_To_025.cs [hunk 2]
```diff
@@ -141,11 +141,29 @@ private static bool IsDivisibleBy1Through20(int number)
...
+            {
+                sumOfSquares += i * i;
+            }
+
+            long sumOfFirsthundredNums = 0;
+
+            for(int index = 1; index <= 100; index++)
+            {
+                sumOfFirsthundredNums += index;
```
Comment (zhollis21) [inline ProjectEuler/Problems/Problems_001_To_025.cs]: your variable for the sum of 100 numbers should have a capital 'H' for Hundred

**Review by zhollis21 — CHANGES_REQUESTED** [review]
Code works great. Just some minor performance and naming changes.

File: ProjectEuler/Problems/Problems_001_To_025.cs (hunk 1 above)
Comment (paulaOliverosHollis) [reply ProjectEuler/Problems/Problems_001_To_025.cs]: I see what you mean. I fixed it.

File: ProjectEuler/Problems/Problems_001_To_025.cs (hunk 2 above)
Comment (paulaOliverosHollis) [reply ProjectEuler/Problems/Problems_001_To_025.cs]: I missed that. I fixed it. Thanks!

**Review by zhollis21 — APPROVED** [review]

**Pull request merged**